systemd/adbd-prepare.service

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2013 Jolla Oy
+#
+# Prepare functionfs for adbd 
+ 
+[Unit]
+Description=functionfs setup for adbd
+After=init-done.service
```

Comment:
This is too late won't work in case of adb is set autoconnection, should be Before=adbd.service PartOf=adbd.service?